infra/base-images/base-clang/checkout_build_install_llvm.sh

```diff
@@ -29,7 +29,7 @@ function clone_with_retries {
   set +e
   for i in $(seq 1 $CHECKOUT_RETRIES); do
     rm -rf $LOCAL_PATH
-    git clone $REPOSITORY $LOCAL_PATH
```

Comment:
I'm pretty certain this breaks.

Reply:
@jonathanmetzman  not sure why you think this break and not needed . With "--depth" which is a flag of "git" command if git command runs successfully why it will fail with its flag .

Also only clone the level with depth defined , and it is useful in term of speed and time required for cloning big repos with long histories . 

Using this , clone time reduce , which results in faster build or fuzzing of projects.

Reply:
Because later code in this script can't work if `--depth 1` is used, it needs the commits in the repo it clones.
I don't think the faster time we get from cloning is worth the time it will take to find all such cases where `--depth 1` breaks things.